config/config-matterwick.default.json

```diff
@@ -49,6 +49,8 @@
       "CWSCloudURL": "",
       "CWSStripeKey": "",
       "CWSCloudDNSDomain": "",
+      "CWS_LICENSE_GENERATOR_URL": "",
+      "CWS_LICENSE_GENERATOR_KEY": "",
```

Comment:
this should be `"CWSLicenseGeneratorKey": "",`  and similar to the above

Reply:
@jwilander @gigawhitlocks

Reply:
fuck, obviously, my mistake @cpanato